pytorch_lightning/loops/optimizer/optimizer_loop.py

```diff
@@ -355,22 +330,3 @@ def _training_step(
         # the loss will get scaled for amp. avoid any modifications to it
         loss = closure_loss.detach().clone()
         return AttributeDict(closure_loss=closure_loss, loss=loss, result_collection=result_collection)
-
-    def _track_and_norm_grad(self, optimizer: torch.optim.Optimizer) -> Dict[str, float]:
-        """Tracks gradient norms and clips the gradients of all parameters optimized by the current optimizer.
-
-        Args:
-            optimizer: the current optimizer
-        """
-        # track gradient norms
-        grad_norm_dict = {}
-        can_log = (self.trainer.global_step + 1) % self.trainer.log_every_n_steps == 0
```

Comment:
I don't know why we were checking `can_log`. This is a responsibility of the logger connector.

Perhaps just for increased efficiency?